tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalRuntime.java

```diff
@@ -223,7 +265,56 @@ public void propertyChange( PropertyChangeEvent evt )
     @Override
     public IStatus validate()
     {
+        IStatus status = super.validate();
+
+        if( !status.isOK() )
+        {
+            return status;
+        }
+
+        if ( portalBundle == null)
+        {
+            return new Status( IStatus.ERROR, LiferayServerCore.PLUGIN_ID, 0, Msgs.errorPortalNotExisted, null );
+        }
+
+        if( !portalBundle.getVersion().startsWith( "7" ) )
+        {
+            return new Status( IStatus.ERROR, LiferayServerCore.PLUGIN_ID, 0, Msgs.errorPortalVersion70, null );
+        }
+
+        if( getVMInstall() == null )
+        {
+            return new Status( IStatus.ERROR, LiferayServerCore.PLUGIN_ID, 0, Msgs.errorJRE, null );
+        }
+
+        if( portalBundle.getVersion().startsWith( "7" ) )
+        {
+            IVMInstall vmInstall = getVMInstall();
+            
+            if( vmInstall instanceof IVMInstall2 )
+            {
+                String javaVersion = ( (IVMInstall2) vmInstall ).getJavaVersion();
+                
+                if( javaVersion != null && !isVMMinimumVersion( javaVersion, 106 ) )
```

Comment:
The minimum version is java 1.7, so this check needs to be a little clearer, making the check against "106" makes it seem that version 1.6 is minimum required version.  Please rework this part some.